sw/device/rom_exts/manifest.hjson

```diff
@@ -0,0 +1,156 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  fields : [
+    { name: "manifest_identifier",
+      desc: '''ROM_EXT Manifest Identifier.
+
+            TODO
+            ''',
+      size: "32",
```

Comment:
Please don't use strings where you can use numbers.
```suggestion
      size: 32,
```

Reply:
That is how `reggen` does it, I am happy to try it out though.

Reply:
Done.